sap-schemas/v3.0/schema.json

```diff
@@ -78,7 +78,24 @@
                   "type": "string"
                 },
                 {
-                  "type": "object"
+                  "type": "object",
+                  "additionalProperties": false,
+                  "required": ["text", "format"],
+                  "properties": {
+                    "text": {
+                      "type": "string",
+                      "description": "The actual value for a given extension."
+                    },
+                    "key": {
+                      "type": "string",
+                      "description": "The value key"
+                    },
```

Comment:
Let's not merge this until we agree with S/4 on the approach.

But we can merge this PR to better document the status quo if we leave this out?